src/main/java/org/apache/sling/distribution/serialization/impl/vlt/VaultDistributionPackageBuilderFactory.java

```diff
@@ -295,6 +295,7 @@ public void activate(BundleContext context, Map<String, Object> config) {
             Dictionary<String, Object> props = new Hashtable<String, Object>();
             props.put(Scheduler.PROPERTY_SCHEDULER_CONCURRENT, false);
             props.put(Scheduler.PROPERTY_SCHEDULER_PERIOD, cleanupDelay);
+            props.put(Scheduler.PROPERTY_SCHEDULER_RUN_ON, Scheduler.VALUE_RUN_ON_LEADER);
```

Comment:
how about?
```suggestion
            props.put(Scheduler.PROPERTY_SCHEDULER_RUN_ON, Scheduler. VALUE_RUN_ON_SINGLE);
```

Reply:
SINGLE works as well, we don't have to run this on the leader. Updated.